redaxo/src/addons/structure/plugins/content/lib/article_content.php

```diff
@@ -62,10 +62,15 @@ protected function _getValue($value)
     }
 
     public function hasValue($value)
+    {
+        return $this->_hasValue($value);
+    }
+
+    public function _hasValue($value)
     {
         // bc
         if ($this->viasql) {
-            return parent::hasValue($value);
+            return parent::_hasValue($value);
```

Comment:
Muss es hier nicht in `hasValue` gehen, sonst greift doch hier die Präfixergänzung nicht, oder?

Reply:
äh stimmt.
Arrg, stimmt die unit tests checken auch nicht den bc case. Ergänze ich dann nochmal.

Reply:
@gharlan wozu ist denn der bc `viasql` ? Das war schon als die Klasse angelegt wurde initial drin. Hat das überhaupt noch einen zweck?

Reply:
Das wurde in diesem Refactoring eingebaut (rex_article aufklappen, da ist es drin, so hieß ja früher die jetzige rex_article_content klasse): https://github.com/redaxo/redaxo4/commit/460e0ae472417e1551801aacb0de1ab17cbaa786#diff-05e8bfecf9d2b2c681cf516856329de6R89-R90

Also noch zu R4-Zeiten, und ich weiß es auch nicht so wirklich, wofür es gebraucht wird/wurde.

Ich meine, dass es auch mal in einem Frontend-Editing-Ansatz Verwendung fand, der wurde aber ja nie abgeschlossen.

Reply:
Vllt separates issue aufmachen wo die frage gestellt wird ob das weg kann?

Reply:
Oder es einfach drin lassen ;)

Hier im PR sollte es jedenfalls passen, wenn du die Zeile in hasValue änderst.